packages/@aws-cdk/aws-glue/README.md

```diff
@@ -173,6 +190,12 @@ new glue.Database(this, 'MyDatabase', {
   databaseName: 'my_database',
 });
 ```
+The Glue Catalog is populated with data using Glue Crawlers. For more information on crawlers, see [CfnCrawler](https://docs.aws.amazon.com/cdk/api/v1/docs/@aws-cdk_aws-glue.CfnCrawler.html) construct.
+
+There are some limitations between Crawlers, `Databases`, and `Tables`:
```

Comment:
>some limitations

[nit] Unless the limitations are contained within the same package/ service/ area of concern, I'd say place references, that way this file does not have to get updated when other docs do.